### ISSUE TITLE
Local fallback asset wrong

### ISSUE THREAD

**Issue by joakimfischer** [andrewdavey/cassette#204]
Hi,

I use your suggested tecnique for local fallback (http://getcassette.net/documentation/scripts/external) but I think there is a small error with fallbackurl.. the scripts write out the CDN instead of the local assett in the fallback script. See below example. In the last box you can see the CDN instead of the local assett.

``` c#
Bundles.Reference("scripts/fallback/jquery.js");
```

``` c#
bundles.AddUrlWithLocalAssets(
                "//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js",
                new LocalAssetSettings
                {
                    // Local assets for use in debug mode.
                    Path = "scripts/fallback/jquery.js",
                    // When condition is true, uses the local assets
                    FallbackCondition = "!window.jQuery"
                }
            );
```

``` javsacript
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
<script type="text/javascript">
if(!window.jQuery){
document.write(unescape('%3Cscript src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"%3E%3C/script%3E'));
}
</script>
```

**Comment by andrewdavey** [comment]
Thanks for catching this bug. I've fixed it in `master` and will get a new release out soon.

**Comment by joakimfischer** [comment]
I would have fix the error myself but I haven't gotten started with GitHub yet. I have all my own projects in subversion and not public but I would like the change this, as well as getting started with nuget as well :)

**Comment by joakimfischer** [comment]
Sorry. Maybe I should not close it until its releases.

**Comment by andrewdavey** [comment]
No worries :) If you do start digging into Cassette's code and need any git or nuget help just let me know.

**Comment by andrewdavey** [comment]
Fixed in v1.2.0
